addons/account/models/account_payment.py

```diff
@@ -120,7 +120,7 @@ def default_get(self, fields):
         if not active_model or not active_ids:
             raise UserError(_("Programmation error: wizard action executed without active_model or active_ids in context."))
         if active_model != 'account.invoice':
-            raise UserError(_("Programmation error: the expected model for this action is 'account.invoice'. The provided one is '%d'.") % active_model)
+            return rec
```

Comment:
please add a comment saying why we may face another active model than account.invoice